dffbackend/Models/Signature.cs

```diff
@@ -0,0 +1,23 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace dffbackend.Models;
+
+public class Signature
+{
+    [Key]
+    public Guid Id { get; set; }
+
+    [Required]
+    public Guid FactoringCompanyId { get; set; }
+
+    [Required]
+    public FactoringCompany FactoringCompany { get; set; }
+
+    public string Signature1 { get; set; }
+    
+    public string Signature2 { get; set; }
+
+    public string Signature3 { get; set; }
+
+    public string Signature4 { get; set; }
```

Comment:
Create issue to add IsDuplicate boolean so that we can mark them during the check and query them easier later on.

Reply:
maybe it should be a separate table that would reference this one; how will we know which signature (1, 2, 3 or 4) from a single signature db entry was duplicate?